scripts/add_pools.py

```diff
@@ -31,15 +31,18 @@ def add_pool(data, registry, deployer):
     if "base_pool" in data:
         # adding a metapool
         registry.add_metapool(
-            swap, n_coins, token, decimals, {"from": deployer, "gas_price": gas_strategy}
+            swap, n_coins, token, decimals, pool_name, {"from": deployer, "gas_price": gas_strategy}
         )
         return
 
     is_v1 = data["lp_contract"] == "CurveTokenV1"
     has_initial_A = hasattr(swap, "intitial_A")
-    rate_method_id = "0x00"
+    rate_info = "0x00000000"
     if "wrapped_contract" in data:
-        rate_method_id = RATE_METHOD_IDS[data["wrapped_contract"]]
+        rate_info = RATE_METHOD_IDS[data["wrapped_contract"]]
+    if "rate_calculator_address" in data:
+        # 24-bytes = 20-byte address + 4-byte fn sig
+        rate_info = data["rate_calculator_address"] + rate_info[2:]
```

Comment:
I'll have to update the curve-contract repo's pool-templates documentation to include this information. It made sense to source the address from the pool-data.json instead of adding the the calculator contract address directly to this `add_pool` script similar to the `RATE_METHOD_IDS` global variable, because it bundles pool specific contracts in the curve-contract repository.

Reply:
If this code ever makes it on-chain, will also need to add a calculator for the ankerETH pool